src/main/java/com/faforever/client/patch/GameBinariesUpdateTaskImpl.java

```diff
@@ -107,11 +110,16 @@ private void downloadFafExeIfNecessary(Path exePath) throws IOException {
             .listener(this::updateProgress)
             .copy();
       }
+      makeSureExeIsExecutable(exePath);
     } finally {
       ResourceLocks.freeDownloadLock();
     }
   }
 
+  private void makeSureExeIsExecutable(Path exePath) {
```

Comment:
we're not making sure the executable is executable (not checking it before), we're just setting the executable bit (rename method pls)